gammapy/data/pointing.py

```diff
@@ -131,6 +131,8 @@ def mode(self):
         obs_mode = self.meta.get("OBS_MODE")
         if obs_mode is None:
             return PointingMode.POINTING
+        if np.size(obs_mode)>1 and "POINTING" in [i.upper() for i in obs_mode]:
```

Comment:
Hm, the need for this statement is not clear to me. In any case one should avoid the hard-coded "POINTING" string here. This is the whole point of the existence of the `PointingMode` enum object. I'll comment in more detail further down...

Reply:
Maybe this should go in `PointingMode.from_gadf_string` instead?

Reply:
well I added that because the data used in the `spectral_analysis_rad_max` tutorial have `OBS_MODE = ["POINTING", "WOBBLE"]`, which gives an error because `PointingMode.from_gadf_string` expects a single string and not a list...

Reply:
but indeed I can move that logic inside `PointingMode.from_gadf_string`

Reply:
it would also work to take the first of the entries of the list... I just thought that checking if `POINTING` was in it would be best....

Reply:
> well I added that because the data used in the spectral_analysis_rad_max tutorial have OBS_MODE = ["POINTING", "WOBBLE"], which gives an error because PointingMode.from_gadf_string expects a single string and not a list...

Is having a list of pointing modes GADF compliant? Do you know?

Reply:
technically no - GADF specifies it should be a string

Reply:
but `OBS_MODE` is also not mandatory in 0.2... so it is a bit of a gray area

Reply:
OK, maybe @cosimoNigro can shortly comment. I think he produced the rad max files...